portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,84 @@
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: 80vw;
```

Comment:
Google's CSS style guide recommends alphabetizing CSS rules: https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

I actually personally don't like this rule, so it's up to you whether you want to follow it or not.

Reply:
I alphabetized everything in css file, I left the spacing as is, because it is more readable for me, but I could change that as well.